packages/amplify_auth_plugin_interface/lib/src/Session/AuthUserAttribute.dart

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:flutter/foundation.dart';
+
+
+class AuthUserAttribute {
```

Comment:
The naming of the files and directories are against the best practices of dart. https://dart.dev/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores

But this is something we haven't followed throughout the library. Should we pick this up pre-release (renaming all the files)?